app/controllers/idv/hybrid_mobile/document_capture_controller.rb

```diff
@@ -87,6 +88,10 @@ def redo_document_capture_pending?
 
         document_capture_session.requested_at > stored_result.captured_at
       end
+
```

Comment:
As shared in slack, I tested the hybrid flow using `/test/telephony` and was able to successfully update the state on the "Update your mailing address" page.  Good work, @WilliamBirdsall.